### ISSUE TITLE
Is there any config to ignore all test files?

### ISSUE THREAD

**Issue by zhuxubin01** [mgechev/revive#645]
I want to ignore all test files. Is there any way to do it?

**Comment by chavacava** [comment]
Hi @zhuxubin01, sorry for the late response.
There is no an explicit flag to ignore test files.

**Comment by chavacava** [comment]
If your goal is to do not receive lint failures related to test files you could use `grep` to filter out the output, something like:
```bash
$ revive | grep -v _test.go
```

**Comment by mgechev** [comment]
Closing for now.
